pages/common/quarkus.md

```diff
@@ -0,0 +1,36 @@
+# quarkus
+
+> Create projects, manage extensions and perform essential build and development tasks.
+> More information: <https://quarkus.io/guides/cli-tooling>.
+
+- Create a new application project:
+
+`quarkus create app {{project_name}}`
+
+- Run the current project in live coding mode:
+
+`quarkus dev`
+
+- Run the application:
+
+`quarkus run`
+
+- Run the current project in continuous testing mode:
+
+`quarkus test`
+
+- Add extension(s) to the current project:
+
+`quarkus extension add {{extension_name}}`
```

Comment:
```suggestion
`quarkus extension add {{extension_1 extension_2 ...}}`
```

If I got it right, multiple extesions can be specified, right?

Also, do the extensions have some kind of ID or anything like that which makes "extension1" ambigous or not? If so, we can go with:

```suggestion
`quarkus extension add {{extension_name_1 extension_name_2 ...}}`
```